spec/lang/step/terminators.md

```diff
@@ -325,7 +328,7 @@ impl<M: Memory> Machine<M> {
             ReturnAction::BottomOfStack => {
                 // Only the bottom frame in a stack has no caller.
                 // Therefore the thread must terminate now.
-                self.terminate_active_thread()?;
+                self.terminate_active_thread("return")?;
```

Comment:
I'm not a fan of passing around `&str` arguments.

I think it's fine here to just have the error say "the start function must not return nor unwind".